pipeline/acquisition.py

```diff
@@ -0,0 +1,375 @@
+#    photo_stim_wavelength: int
+#    photo_stim_method: enum('fiber', 'laser')
+#    -> reference.BrainLocation.proj(photo_stim_location="brain_location")
+#    -> reference.Hemisphere.proj(photo_stim_hemisphere="hemisphere")
+#    -> reference.CoordinateReference.proj(photo_stim_coordinate_ref="coordinate_ref")
+#    photo_stim_coordinate_ap: float    # in mm, anterior positive, posterior negative 
+#    photo_stim_coordinate_ml: float    # in mm, always postive, number larger when more lateral
+#    photo_stim_coordinate_dv: float    # in mm, always postive, number larger when more ventral (deeper)
+#    """
+
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    -> subject.Subject
+    session_time: datetime    # session time
+    ---
+    session_directory = "": varchar(256)
+    session_note = "" : varchar(256) 
+    """
+
+    class Experimenter(dj.Part):
+        definition = """
+        -> master
+        -> reference.Experimenter
+        """
+
+    class ExperimentType(dj.Part):
+        definition = """
+        -> master
+        -> ExperimentType
+        """
+
+
+@schema
+class IntracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the intracelluar recording (e.g. cell info)
+    -> Session
+    cell_id: varchar(36) # a string identifying the cell in which this intracellular recording is concerning
+    ---
+    cell_type: enum('excitatory','inhibitory','N/A')
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+    
+@schema
+class ExtracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the extracelluar recording (e.g. location)
+    -> Session
+    ec_id: varchar(36) # extracellular_id: a string uniquely identify this extracellular recording routine of this recording session (there might be multiple extracellular recording in 1 recording session, at different location for example)
+    ---
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+    
+@schema
+class StimulationInfo(dj.Manual):
+    definition = """ # Table containing information relating to the stimulatiom (stimulation type (optical or electrical), location, device)
+    -> Session
+    stim_id: varchar(36) # stimulation_id: a string uniquely identify this stimulation routine of this recording session (there might be multiple stimulation of the same or different type (phototim or electrical) in 1 recording session)
+    ---
+    stim_type: enum('optical','electrical')
+    -> ActionLocation
+    -> reference.Device
+    """    
+
+
+@schema
+class TrialSet(dj.Imported):
+    definition = """
+    -> Session
+    ---
+    n_trials: int # total number of trials
+    """
+    
+    class Trial(dj.Part):
+        definition = """
+        -> master
+        trial_id: varchar(36)           # unique id of this trial in this trial set
+        ---
+        cue_start_time: float           # cue onset of this trial, with respect to this trial's start time
```

Comment:
You should denote in comments the units (ms, s, etc) used for measurements unless it's obvious. In this case, I won't be sure if things are measured in seconds or milliseconds.